salvo/BUILD

```diff
@@ -0,0 +1,14 @@
+load("@rules_python//python:defs.bzl", "py_binary", "py_library")
+
+licenses(["notice"])  # Apache 2
+
+py_binary(
+    name = "salvo",
+    srcs = ["src/salvo.py"],
```

Comment:
As discussed offline, the move of `salvo.py` into the `src/` directory was likely caused by a mistake I made in one of my comments. If we can, let's move it back to the `salvo/` directory.

Reply:
I had a merge conflict and I didn't catch this.  I'll fix and update